rocketc/rocketc.py

```diff
@@ -343,16 +353,30 @@ def _update_user(self, user_id, username, email):
         self._set_avatar(username)
 
     @XBlock.json_handler
-    def set_default_channel(self, data, suffix=""):
+    def create_group(self, data, suffix=""):
```

Comment:
The "suffix" paramater is always required even when it's not used?

Reply:
yes, the methods with json_handler decorator always needs data and suffix